2/contrib/jenkins/kube-slave-common.sh

```diff
@@ -31,6 +31,10 @@ export JNLP_PORT=${JNLP_PORT:-50000}
 export NODEJS_SLAVE=${NODEJS_SLAVE_IMAGE:-image-registry.openshift-image-registry.svc:5000/openshift/jenkins-agent-nodejs:latest}
 export MAVEN_SLAVE=${MAVEN_SLAVE_IMAGE:-image-registry.openshift-image-registry.svc:5000/openshift/jenkins-agent-maven:latest}
 
+export AGENT_BASE=${AGENT_BASE_IMAGE:-image-registry.openshift-image-registry.svc:5000/openshift/jenkins-agent-base:latest}
+export JAVA_SIDECAR=${JAVA_SIDECAR_IMAGE:-image-registry.openshift-image-registry.svc:5000/openshift/java:latest}
```

Comment:
same here

Reply:
I would use JAVA_BUILDER
```suggestion
export JAVA_BUILDER=${JAVA_SIDECAR_IMAGE:-image-registry.openshift-image-registry.svc:5000/openshift/java:latest}
```